pwa-backend/src/test/java/io/growingabit/testUtils/BaseDatastoreTest.java

```diff
@@ -1,14 +1,17 @@
 package io.growingabit.testUtils;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+
 import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalMailServiceTestConfig;
 import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalTaskQueueTestConfig;
 import com.googlecode.objectify.ObjectifyFactory;
 import com.googlecode.objectify.ObjectifyService;
 import com.googlecode.objectify.cache.AsyncCacheFilter;
 import com.googlecode.objectify.util.Closeable;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
 
 public class BaseDatastoreTest {
```

Comment:
A questo punto rinominiamola in `BaseGaeTest`

Reply:
ok